diff --git a/node_modules/@astro-community/astro-embed-youtube/YouTube.astro b/node_modules/@astro-community/astro-embed-youtube/YouTube.astro
index 1e43228..3768fb8 100644
--- a/node_modules/@astro-community/astro-embed-youtube/YouTube.astro
+++ b/node_modules/@astro-community/astro-embed-youtube/YouTube.astro
@@ -19,7 +19,7 @@ function extractID(idOrUrl: string) {
 }
 
 const videoid = extractID(id);
-const posterURL = poster || `https://i.ytimg.com/vi/${videoid}/hqdefault.jpg`;
+const posterURL = poster || `https://i.ytimg.com/vi/${videoid}/hq720.jpg`;
 
 // TODO: use the progressive enhancement pattern once https://github.com/paulirish/lite-youtube-embed/issues/124 is released.
 ---
@@ -27,7 +27,7 @@ const posterURL = poster || `https://i.ytimg.com/vi/${videoid}/hqdefault.jpg`;
 <lite-youtube
 	{videoid}
 	{...attrs}
-	style={`background-image: url('${posterURL}');`}
+	style={`background-image: url('${posterURL}'); margin-bottom: 2rem;`}
 />
 
 <script>
